CHANGELOG.md

```
@@ -1,3 +1,8 @@
+## 0.13.0
+
+#### New icons
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)